ecr/fetcher.go

```diff
@@ -200,7 +201,7 @@ func (f *ecrFetcher) fetchLayerHtcat(ctx context.Context, desc ocispec.Descripto
 		_, err := htc.WriteTo(pw)
 		if err != nil {
 			log.G(ctx).
-				WithError(err).
+				WithError(httputil.RedactHTTPQueryValuesFromError(err)).
 				WithField("url", downloadURL).
```

Comment:
The download URL here is sensitive. I think that actually exists in several spots in this file.